flutter_appauth_platform_interface/lib/src/token_request.dart

```diff
@@ -15,7 +15,8 @@ class TokenRequest with CommonRequestDetails {
       String discoveryUrl,
       this.authorizationCode,
       this.codeVerifier,
-      bool allowInsecureConnections = false})
+      bool allowInsecureConnections = false,
+      bool preferEphemeralSession = false})
```

Comment:
Apologies as I missed this last time but  believe this class should have the changes reverted as the parameter has no effect for a token request right?

Reply:
@MaikuB You are correct. I've removed it from TokenRequest

Reply:
Thanks though the platform interface tests are failing at the moment. This can be solved by moving the field from the `CommonRequestDetails` to the `AuthorizationParameters` instead. I tried to make the changes and push them but it failed. I'm guessing this is because the setting to allow for maintainers to make changes to this PR wasn't enabled